docs/resie_energy_system_components.md

```diff
@@ -594,13 +594,93 @@ Modelled as [fuel boiler](resie_energy_system_components.md#fuel-boiler-fb) with
 ## Biomass boiler (BB)
 Modelled as [fuel boiler](resie_energy_system_components.md#fuel-boiler-fb) with solid biomass fuel as input.
 
-## Heat sources 
-### Soil
-- geothermal probes --> see below
-- geothermal collector --> see below
-- geothermal basket collector --> not included for now
-- geothermal trensh collector --> not included for now
-- geothermal spiral collector --> not included for now
+
+## Heat exchanger
+![Energy flow of a heat exchanger](fig/240613_heat_exchanger.svg)
+
+There are many different types of heat exchangers, all of which result in different calculations when derived from first principles of thermodynamics. As detailed thermodynamics and hydraulics are not part of ReSiE's simulation model, a simplified model is used. Of the three inputs \(T_{source,in}\), \(T_{sink,in}\), \(\dot{Q}_{in}\) and three outputs \(T_{source,out}\), \(T_{sink,out}\), \(\dot{Q}_{out}\), typically \(T_{source,out}\) and \(T_{sink,in}\) remain unknown. Here, "source" and "sink" stand in for the commonly called hot and cold sides of a heat exchanger.
+
+**Approximation of temperature reduction**
+
+Given the unknown variables and the number of heat exchanger variations, no definitive calculation of \(T_{sink,out}\), which is the important variable for further calculations of other components, seems possible. The simplest solution is a reduction by a fixed amount (which might be zero). A more complicated approach is based on the logarithmic mean temperature difference (LMTD), which is defined as
+
+$${LMTD} = \frac{\Delta T_A - \Delta T_B}{\ln{\Delta T_A} - \ln{\Delta T_B}}$$
+$$\Delta T_A = T_{source,in} - T_{sink,out}$$
+$$\Delta T_B = T_{source,out} - T_{sink,in}$$
+
+for a counter-current heat exchanger. Similar calculations can be made for cocurrent and cross-current heat exchangers. Setting \(\Delta T_B = \alpha \cdot \Delta T_A\) allows us to disregard the unknown variables by introducing a new one in the form of \(\alpha\), which is a measure for the temperature spread across the length of the exchange area. As heat exchangers are designed for a specific range in temperatures and mass flow, \(\alpha\) is expected to change in respect to the temperatures, becoming smaller the more \(T_{source,in}\) approaches the extreme sides of the range.
+
+This leads to the following formulation:
+
+$$T_{source,in,avg} = \frac{1}{2} \cdot (T_{source,in,max} + T_{source,in,min})$$
+$$\alpha(T_{source,in}) = 1 - \frac{abs(T_{source,in,avg} - T_{source,in})}{(T_{source,in,max} - T_{source,in,min})}$$
+$$\alpha_c = min(\alpha_{max},\ \alpha(T_{source,in}))$$
+$$T_{sink,out} = T_{source,in} - L_{min} \cdot \frac{\ln{\alpha_c^{-1}}}{1-\alpha_c}$$
```

Comment:
What is L_{min}?

Reply:
A scaling factor that is essentially the minimum reduction in temperature when alpha is maximal (close to the average temperature). For a value of L_min = 2.0 K, at the average temperature the source input temperature is reduced by 2.11 K.